workers/unity/Packages/com.improbable.gdk.core/Config/ReceptionistConfig.cs

```diff
@@ -48,5 +48,16 @@ public static ReceptionistConfig CreateConnectionConfigForPhysicalAndroid(string
                 UseExternalIp = true
             };
         }
+
+        public static ReceptionistConfig CreateConnectionConfigForiOS()
+        {
+            return new ReceptionistConfig
+            {
+                // By default this config only connects to local IP (127.0.0.1)
+                // Uncomment the following line with hardcoded IP to connect to non-local deployment
+                // ReceptionistHost = "Your IP address",
```

Comment:
it's already merged in, but we shouldn't have commented out code like this. Also, there is no need to hardcode the IP for a non-local deployment. If you are connecting to a cloud deployment, you won't use the Receptionst flow, but the Locator flow.

Reply:
Sorry, poor wording. This is actually a convenience shortcut for device client to be able to connect to any deployment - for now we only have workaround for input field crash.
I am happy to remove this if you think we shouldn't have the code commented out